packages/valory/skills/staking_abci/behaviours.py

```diff
@@ -97,11 +97,6 @@ def mech_activity_checker_contract(self) -> str:
         """Get the staking contract address."""
         return self.params.mech_activity_checker_contract
 
-    @staking_contract_address.setter
-    def staking_contract_address(self, staking_contract_address: str) -> None:
-        """Set the staking contract address."""
-        self.params.staking_contract_address = staking_contract_address
-
```

Comment:
@Adamantios was this function ever used? I think it is incorrect.

Reply:
Not used.